05/app.js

```diff
@@ -1,14 +1,47 @@
 const stats = {
     paragraphs: {
         'p1': 0,
+        'p2': 0,
+        'p3': 0,
```

Comment:
Nie należało tego dopisywać ręcznie - jeśli będzie tych elementów będzie 100 lub będą się zmieniać co godzinę to będziesz je dopisywać? Pewnie, że nie! Trzeba to zrobić "dynamicznie"